docs/documentation/server_admin/topics/authentication/webauthn.adoc

```diff
@@ -19,71 +19,45 @@ WebAuthn's specification uses a `user.id` to map a public key credential to a sp
 The setup procedure of WebAuthn support for 2FA is the following:
 
 [[_webauthn-register]]
-===== Enable WebAuthn authenticator registration
+===== Check WebAuthn authenticator registration is enabled
 
 . Click *Authentication* in the menu.
 . Click the *Required Actions* tab.
-. Toggle the *Webauthn Register* switch to *ON*.
+. Check action *Webauthn Register* switch is set to *ON*.
 
 Toggle the *Default Action* switch to *ON* if you want all new users to be required to register their WebAuthn credentials.
 
 [[_webauthn-authenticator-setup]]
-==== Adding WebAuthn authentication to a browser flow
+==== Enable WebAuthn authentication in the default browser flow
 
 . Click *Authentication* in the menu.
 . Click the *Browser* flow.
-. Select *Duplicate* from the "Action list" to make a copy of the built-in *Browser* flow.
-. Enter "WebAuthn Browser" as the name of the copy.
-. Click *Duplicate*.
-. Click the name to go to the details
-. Click the trash can icon 🗑️ of the "WebAuthn Browser Browser - Conditional OTP" and click *Delete*.
-
-If you require WebAuthn for all users:
-
-. Click *+* menu of the *WebAuthn Browser Forms*.
-. Click *Add step*.
-. Click *WebAuthn Authenticator*.
-. Click *Add*.
-. Select *Required* for the *WebAuthn Authenticator* authentication type to set its requirement to required.
+. Locate the execution *WebAuthn Authenticator* inside the *Browser - Conditional 2FA* sub-flow.
+. Change the _requirement_ from _Disabled_ to _Alternative_ for that execution.
 +
-image:images/webauthn-browser-flow-required.png[Webauthn browser flow required]
-+
-. Click the *Action* menu at the top of the screen.
-. Select *Bind flow* from the drop-down list.
-. Select *Browser* from the drop-down list.
-. Click *Save*.
+.WebAuthn browser flow conditional with OTP
+image:images/webauthn-browser-flow-conditional-with-OTP.png[WebAuthn browser flow conditional with OTP]
 
-[NOTE]
-====
-If a user does not have WebAuthn credentials, the user must register WebAuthn credentials.
-====
+With this configuration, the users can choose between using WebAuthn and OTP for the second factor. As the sub-flow is _conditional_, they are only asked to present a 2FA credential (OTP or WebAuthn) if they have already registered one of the respective credential types. If a user has configured both credential types, the credential with the highest priority will be displayed by default, but another option *Try Another Way* will be available that allows to select the other 2FA method.
```

Comment:
```suggestion
With this configuration, the users can choose between using WebAuthn and OTP for the second factor. As the sub-flow is _conditional_, they are only asked to present a 2FA credential (OTP or WebAuthn) if they have already registered one of the respective credential types. If a user has configured both credential types, the credential with the highest priority will be displayed by default. However, the  *Try Another Way* option will appear so that the user has an alternative to log in.
```